src/utils/MultiInviter.ts

```diff
@@ -170,6 +172,27 @@ export default class MultiInviter {
                     errcode: USER_ALREADY_INVITED,
                     error: "Member already invited",
                 });
+            } else if (member?.membership === "ban") {
+                let proceed = false;
+                const ourMember = room.getMember(this.matrixClient.getSafeUserId());
+                if (!!ourMember && room.currentState.hasSufficientPowerLevelFor("ban", ourMember.powerLevel)) {
```

Comment:
it might be worth noting that having `ban` permission is not all that is required. Cf https://spec.matrix.org/v1.7/rooms/v10/#authorization-rules 4.5.{4,5}:

> If the sender’s power level is greater than or equal to the kick level, and the target user’s power level is less than the sender’s power level, allow. Otherwise, reject.

In other words: you also need power level for "kick", and to have a higher power level than the target.

I'm not sure if we need to care about these edge-cases: it's a material improvement as it stands. Just wanted to check you were aware.

Reply:
> I'm not sure if we need to care about these edge-cases: it's a material improvement as it stands. Just wanted to check you were aware.

I was not as the C-S API spec is unclear, it just says 

> The caller must have the required power level in order to perform this operation.

Reply:
>  The caller must have the required power level in order to perform this operation.

Well, that's correct, if unhelpful :)

Reply:
Oh for sure :D